app/qml/TextHyperlink.qml

```diff
@@ -0,0 +1,25 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.7
+import QtQuick.Controls 2.2
+import "."  // import InputStyle singleton
+
+Text {
+  property string richText: "(no-text)"
+
+  text: "<style>a:link { color: " + InputStyle.highlightColor
+        + "; text-decoration: underline; }</style>" + richText
+  color: InputStyle.fontColor
+  onLinkActivated: Qt.openUrlExternally(link)
+  textFormat: Text.RichText
+  wrapMode: Text.WordWrap
+  font.pixelSize: InputStyle.fontPixelSizeNormal
+  font.bold: true
+}
```

Comment:
I would suggest to have `Item` with a `text` property and `Text` item inside... otherwise it's easy for caller to use `text` property by default instead of `richText`